services/app-web/src/constants/routes.ts

```diff
@@ -47,6 +48,7 @@ const RoutesRecord: Record<RouteT, string> = {
     HELP: '/help',
     SETTINGS: '/settings',
     RATES_SUMMARY: 'rates/:id',
+    RATE_EDIT: '/:id/edit',
```

Comment:
To me  this route path should be `/rates/:id/edit`.

The ticket wants the URL to be `mc-review.onemac.cms.gove/:rate-id:/edit`. This seems not to follow our pattern elsewhere, like submissions, where we would append the `rates/` to the URL, so it would be like `mc-review.onemac.cms.gove/rate/:rate-id:/edit`. I would check with design to make sure it wasn't an error.